express/properties/workflow.py

```diff
@@ -1,27 +1,235 @@
 from express.properties import BaseProperty
+import os
+import copy
+from typing import List
+from abc import abstractmethod
 
 
-class ExabyteMLPredictWorkflow(BaseProperty):
+class WorkflowProperty(BaseProperty):
     """
-    Exabyte ML predict Workflow property class.
+    Base class for workflow properties extracted in Express
     """
 
     def __init__(self, name, parser, *args, **kwargs):
-        super(ExabyteMLPredictWorkflow, self).__init__(name, parser, *args, **kwargs)
-        self.name = name
+        """
+        Constructor for PyMLTrainAndPredictWorkflow
+
+        Args:
+            name (str): Name of the workflow
+            parser (str): Parser to use with this workflow
+        """
+        super().__init__(name, parser, *args, **kwargs)
+        self.name: str = name
+
+    @property
+    def schema(self):
+        return self.esse.get_schema_by_id("workflow")
+
+    @abstractmethod
```

Comment:
`@abstractmethod` requires that the class be an abstract base class
https://docs.python.org/3/library/abc.html#abc.abstractmethod

Reply:
abstractmethod decorator has been removed